cmd/osbuild-pipeline/main.go

```diff
@@ -23,12 +23,12 @@ type rpmMD struct {
 
 func main() {
 	var imageType string
-	var blueprintArg string
+	var blueprintArg bool
 	var archArg string
 	var distroArg string
 	var rpmmdArg bool
 	flag.StringVar(&imageType, "image-type", "", "image type, e.g. qcow2 or ami")
-	flag.StringVar(&blueprintArg, "blueprint", "", "path to a JSON file containing a blueprint to translate")
+	flag.BoolVar(&blueprintArg, "blueprint", false, "read a blueprint from stdin")
```

Comment:
Hmm, I think it might be still useful to pass blueprint both from a file and stdin. Many tools can use both filename or `-` as "the main input" (`gcc` for example). It shouldn't make the code much more complicated and I can imagine some cases when it would be more convenient.

Reply:
Fair point. Fixed!